modules/AOR_Reports/controller.php

```diff
@@ -76,7 +76,7 @@ protected function action_changeReportPage(){
         if(!empty($this->bean->id)){
             $this->bean->user_parameters = requestToUserParameters();
             //echo $this->bean->build_report_html($offset, true,$group,$tableId);
```

Comment:
@samus-aran this is already removed on 2nd commit. Please review the PR and recreate / merge manually (wrong branch by original user)